code/lib/api/index.js

```diff
@@ -889,6 +888,37 @@ const api = {
         totalSupply,
       }
     },
+  },
+  rpc: {
+    getTransactionHistory: async ({ address, pageSize = 50, pageIndex = 0, fullTx = false }) => {
+      const { data } = await rpcBase.post('', {
+        jsonrpc: '2.0',
+        method: 'eth_getTransactionsHistory', // eth_ method is non-standard, but we still want to use it because it returns normalized addresses and transaction hashes
```

Comment:
Since this is non-standard, we should replace functions relying on this to using web3 contract events instead